src/server/controllers/match.controller.js

```diff
@@ -37,12 +38,13 @@ export function requestMentoring(req, res, next) {
         // TODO: Confirm method whether send mail or send in-app message.
         mailingUtil.sendEmail(mentor.email, mailStrings.REQUEST_SUBJECT,
           mailStrings.REQUEST_HTML, matchData.contents);
+        pushUtil.sendPush(req.user._id, mentor._id, 'REQUEST', req.user.name);
         return match.save();
       } else {
         throw new Error(matchCallback.ERR_CANNOT_FOUND_MENTOR);
       }
     })
-    .then(() => {
+    .then(match => {
```

Comment:
Oh even one element? Until now we just used it so in back-end code, there's many unfixed forms of them... What you guys think about this?? @lyw0149 @KyuaKwon